### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ShirleyDenkberg**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
(https://github.com/demisto/etc/issues/43786)

## Description
Removed Multi-Factor Authentication from the integration instructions. Added note to use EWS O365, O365 Outlook Mail (Using Graph API), or O365 Outlook Mail Single User (Using Graph API) instead.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by glicht** [comment]
Is there a related issue in etc on this?

I think multi factor note about api key is still relevant for on perm exchange deployments.

**Comment by Bargenish** [comment]
@glicht 
https://github.com/demisto/etc/issues/43786

I'll look into MS docs and will update here.

File: Packs/EWS/Integrations/EWSv2/README.md
Comment (yaakovi) [inline Packs/EWS/Integrations/EWSv2/README.md:4]: you can't use MD format for the links, it should be in HTML format

**Comment by Bargenish** [comment]
Scheduled a call with @Tomer-M on Sunday to test it and check the flow.

File: Packs/EWS/Integrations/EWSv2/README.md
Comment (ShirleyDenkberg) [reply Packs/EWS/Integrations/EWSv2/README.md]: Fixed the formatting.

**Comment by Bargenish** [comment]
Tomer and I hadn't had much luck finding instructions on how to do so.
I have come across the following:
https://docs.microsoft.com/en-us/answers/questions/146959/exchange-server-2016-on-premise-and-2famfa.html

But we still need to verify this

**Comment by ShirleyDenkberg** [comment]
@yaakovi Waiting for your approval.

**Review by yaakovi — APPROVED** [review]

**Pull request merged**
